Apps/Src/Task_LogInfo.c

```diff
@@ -12,7 +13,7 @@ void Task_LogInfo(void *p_arg) {
     EEPROM_LogData(DATA_SOC, Charge_GetPercent());
 
     //1 tick every 10 millisconds, so 300 ticks for a delay of 3000ms --> 3s
-    //OS_OPT_TIME_DLY specifies relative delay, so the delay is started from the �current time + dly�.
+    //OS_OPT_TIME_DLY specifies relative delay, so the delay is started from the �current time + dly�.
```

Comment:
Is it just me or have these quotation marks been changed to diamonds with questions marks in them?

Reply:
I'll look into this